python/doc/examples/reliability_sensitivity/reliability/plot_form_explained.py

```diff
@@ -11,256 +11,307 @@
+# - the Generalized Nataf transformation is used if the input distribution has an elliptical copula,
+# - the Rosenblatt transformation is used in any other cases.
 #
-# Variable transform
-# ^^^^^^^^^^^^^^^^^^
+# The Nataf and  Rosenblatt transformations map the input random vector into a vector that follows a
+# normal distribution with zero mean and unit variance. The Generalized Nataf transformation maps the
+# input random vector into a vector that follows the standard elliptical distribution associated to the
+# elliptical copula of the input distribution.
 #
-
-# %%
-# OpenTURNS has several isoprobabilistic transforms and the FORM/SORM classes implement the Generalized
-# Nataf and Rosenblatt transforms. In this case the `distX` distribution is not elliptical so the default method is the Rosenblatt transform.
+# In this example, the input distribution is not elliptical so the isoprobabilistic transformation is the
+# Rosenblatt transformation.
 #
-print("Is Elliptical ? ", distX.isElliptical())
+print("Is Elliptical ? ", dist_X.isElliptical())
 
 # %%
-# We seek an isoprobabilistic transform :math:`T` such as
+# The Rosenblatt transformation :math:`T` is defined by:
 #
 # .. math::
-#    T : X \mapsto Z
+#    :label: defT
 #
-# where each component of :math:`Z` is a standard unit gaussian.
+#    T : \vect{x} \mapsto \vect{z}
 #
-# The isoprobabilistic transform and its inverse are methods of the distribution `distX` :
-transformation = distX.getIsoProbabilisticTransformation()
-inverseTransformation = distX.getInverseIsoProbabilisticTransformation()
-
-# %%
-# The main goal of this step is to work with a simpler probability density function of
-# the input variables as they will be standard gaussian unit and uncorrelated. The domain of
-# integration will still be complicated but will be handled with a well chosen approximate.
+# such that the random vector :math:`\standardRV = T(\inputRV)` follows a bivariate normal distribution
+# with zero mean and unit variance. It follows that the components :math:`Z_1` and
+# :math:`Z_2` are independent.
 #
-
-# %%
-# We detail the Rosenblatt transform in this simple case.
-# In this example we consider independent variables so the transform is simpler, we only have to
-# perform the transformation on each variable. For the second one is already a standard unit gaussian
-# we transform the first variable only.
+# We detail the Rosenblatt transform in this simple case where the input random vector :math:`\inputRV`
+# has independent components. Then, the Rosenblatt transform is defined by:
+#
+# .. math::
 #
+#    z_i = \Phi^{-1} \circ F_i(x_i)
+#
+# where :math:`F_i` is the cumulative distribution function (CDF) of :math:`X_i` and
+# :math:`\Phi` the CDF of the univariate normal distribution with zero mean and unit variance.
+# Note that in this example, :math:`\Phi^{-1} \circ F_2 = I_d` as :math:`F_2 = \Phi`.
+
+# The isoprobabilistic transform and its inverse are methods of the distribution :
```

Comment:
This is formatted as a comment of the script, which is not correct here:

![image](https://github.com/user-attachments/assets/89313471-5be6-4d1c-ad5c-3b69bafffa0e)

Reply:
yes